pkg/jwt/jwtToken.go

```diff
@@ -0,0 +1,62 @@
+package jwt
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type JwtIssuer struct {
+	issuer string
+	secret string
+	ttl    time.Time
+}
+
+func NewJwtIssuer(extras map[string]interface{}) (*JwtIssuer, error) {
```

Comment:
extras map must be [string]string, I don't remember we could another data type in the MapClaims